modules/UI/avatar/Avatar.js

```diff
@@ -48,6 +48,10 @@ export default {
             users[id] = {};
         }
         users[id][prop] = val;
+        APP.API.notifyAvatarChanged(
+            id === 'local' ? APP.conference.getMyUserId() : id,
```

Comment:
We are setting the id to 'local' if APP.conference.isLocalId(id) is true. At this line we won't be able to check it with APP.conference.isLocalId.

Do you have any refactoring in mind?

Reply:
So you're confident that at this point the local id will always be 'local'?

Reply:
I can't think of a use case where it won't be for the local participant!

Do you have something in mind that worries you?

Reply:
The only concern would be if the id is not "local" because at some point the local user id is set to something other than "local." But if you're confident it will be "local" at this point then the change is fine.

Reply:
I am because we are overriding the id!